content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md

```diff
@@ -377,14 +377,12 @@ Here are some examples of device plugin implementations:
 * The [AMD GPU device plugin](https://github.com/RadeonOpenCompute/k8s-device-plugin)
 * The [Intel device plugins](https://github.com/intel/intel-device-plugins-for-kubernetes) for
   Intel GPU, FPGA, QAT, VPU, SGX, DSA, DLB and IAA devices
-* The [KubeVirt device plugins](https://github.com/kubevirt/kubernetes-device-plugins) for
-  hardware-assisted virtualization
 * The [NVIDIA GPU device plugin for Container-Optimized OS](https://github.com/GoogleCloudPlatform/container-engine-accelerators/tree/master/cmd/nvidia_gpu)
-* The [RDMA device plugin](https://github.com/hustcat/k8s-rdma-device-plugin)
 * The [SocketCAN device plugin](https://github.com/collabora/k8s-socketcan)
-* The [Solarflare device plugin](https://github.com/vikaschoudhary16/sfc-device-plugin)
```

Comment:
Why should we remove this one?

Reply:
It seem the project is not maintained no commit in six years.

Reply:
If you know that this no longer works with Kubernetes (v1.26) then that's grounds to remove it. Otherwise, I'm minded to let readers make up their own mind.